php/libraries/TimePoint.class.inc

```diff
@@ -330,29 +330,39 @@ class TimePoint
 
     /**
      * Sets a piece of data for this timepoint and saves it to the session table.
+     * $newData should be in the form: array($key => $value).
      *
      * This should usually not be used directly but instead use one of the set*
      * methods below. However, there are cases where it may be easier to directly
      * use setData.
      *
-     * @param string $var   The variable to set for this timepoint.
-     * @param string $value The value to set $var to.
+     * @param array $newData Key-value pair(s) of the field(s) to change and its
+     *                              new value(s).
      *
-     * @return void (side-effect modifies session table.)
+     * @return bool true
+     *
+     * @throws DatabaseException
      */
-    function setData($var, $value = null)
+    public function setData(array $newData): bool
```

Comment:
Is this called from anywhere that it can't be made protected?

Reply:
There are 3 instances in `next_stage` that use this function in a static context and are incompatible with `protected` according to phan.